src/SFA.DAS.EmployerFinance.Api/SFA.DAS.EmployerFinance.Api.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>netcoreapp2.2</TargetFramework>
+        <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
```

Comment:
Is this element needed? We're using Kestrel so should be going with the default i.e. `OutOfProcess`.

Reply:
yep you're correct it was left when we didn't use Kestrel. i'll change it now